content/swift/concepts/dictionaries/terms/subscript/subscript.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: 'Subscript'
+Description: 'Accesses the elements of a collection, list or sequence.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+  - 'Computer Science'
+Tags:
+  - 'Dictionary'
+  - 'Properties'
+  - 'Methods'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+**`Subscript`** in Swift is a shortcut to access the elements of a collection, list or sequence. It is a syntactic sugar that allows you to access the elements of a collection using the subscript syntax `[]`. You can define subscripts in your own custom types, and the syntax is similar to computed properties. You can define multiple subscripts with different input parameters and return types.

```

Comment:
```suggestion
**Subscript** in Swift is a shortcut for accessing elements of a collection, list, or sequence. It is syntactic sugar that allows the use of the subscript syntax `[]` to access elements. Subscripts can also be defined in custom types, with syntax similar to computed properties. Multiple subscripts can be defined with different input parameters and return types.

```

Reply:
I personally love this edits. Interplay of active and passive voices. Thanks